src/components/utils/UiButton.vue

```diff
@@ -140,7 +153,7 @@ export default Vue.extend({
     background: var(--color-indicator-medium);
     border: 0;
 
-    &:hover {
+    &:not(^&--disabled):hover {
```

Comment:
А зачем здесь ^? Опечатка или это что-то даёт?
```suggestion
    &:not(&--disabled):hover {
```

Reply:
Разобрался